nltk/tag/perceptron.py

```diff
@@ -140,7 +155,7 @@ def __init__(self, load=True):
             AP_MODEL_LOC = 'file:'+str(find('taggers/averaged_perceptron_tagger/'+PICKLE))
             self.load(AP_MODEL_LOC)
 
-    def tag(self, tokens):
+    def tag(self, tokens, conf=False):
```

Comment:
If we make `predict()` always return confidence then we don't have to propagate the `conf` kwarg =)

Reply:
It's true, but as indicated above, we may want to allow the client to decide, since there's cost.